src/services/Testing/DummyData.js

```diff
@@ -16,7 +16,7 @@ export const dummyItems = {
         id: 212,
         item: 211,
         text: "question 1",
-        type: "subjective",
+        type: "mcq",
```

Comment:
guessing that you needed this as mcq questions will add to scorecard and subjective won't?